en/docs/reference/configurations/configuring-micro-gw-conf-for-docker.md

```diff
@@ -0,0 +1,62 @@
+# Configuring micro-gw.conf for Docker
+
+You can use environment variables when running Docker to override the configurations in the `<MGW-RUNTIME-HOME>/conf/micro-gw.conf` file.
+
+- The `-e` flag in the docker run command is used to create environment variables. 
+- You need to create the variable name by appending the `micro-gw.conf` configuration hierarchy by using an `_` underscore to define each level of the hierarchy.
+
+## Example 1 - Defining the Listener HTTPS port
+
+Let’s assume that you are going to fetch an environment variable for `httpsPort` under the `[listenerConfig]` configuration in the `micro-gw.conf` file. The following code snippet shows part of the configuration in the `micro-gw.conf` file.
```

Comment:
Is it correct to say "going to fetch an environment variable for `httpsPort`going to fetch an environment variable for `httpsPort`"? Aren't we trying to just override the default values as opposed to fetch any value?

Reply:
`Overriding assigned values` appears to be better.

Reply:
Fixed in https://github.com/wso2/docs-mg/pull/168/commits/81bf3c0f68c7d5696ceadf96d8c15d39cab56368